CannibalisticZombies/ProjectSettings/TagManager.asset

```diff
@@ -11,7 +11,7 @@ TagManager:
   - 
   - Water
   - UI
-  - 
+  - interactable
```

Comment:
Correct me if I am wrong, but do you use interactable to identify objects in the code? If so I cannot find it. Since we do not use it, I'd say we can live without the interactable tag for now.